chainer/functions/evaluation/binary_accuracy.py

```diff
@@ -23,7 +26,10 @@ def forward(self, inputs):
         y = y.ravel()
         t = t.ravel()
         c = (y >= 0)
-        return xp.asarray((c == t).mean(dtype='f')),
+        count = (t != self.ignore_label).sum()
+        if int(count) == 0:
```

Comment:
How about `count = xp.maximum(1, (t != self.ignore_label).sum())`?
`int(count)` cause synchronization of GPU and CPU.